### PR TITLE
Add input helper

### PR DESCRIPTION
**Pull request by lytedev**
No idea if you wanted something like this, but here you go!

### CODE REVIEW

**Comment by mhanberg** [comment]
This is dope!

File: lib/advent_of_code/input.ex [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    store_in_cache!(day, year, input)
+    input
+  end
+
+  defp cache_dir do
+    config()
+    |> Keyword.get(
+      :cache_dir,
+      (System.get_env("XDG_CACHE_HOME") || "~/.cache") <> "/advent_of_code_inputs"
```
Comment (mhanberg) [inline lib/advent_of_code/input.ex:60]: Probably better to use `Path.join` here

**Review by mhanberg — APPROVED** [review]

File: lib/advent_of_code/input.ex (hunk 1 above)
Comment (mhanberg) [inline lib/advent_of_code/input.ex:60]:
```suggestion
      Path.join([System.get_env("XDG_CACHE_HOME", "~/.cache"), "/advent_of_code_inputs"])
```

File: mix.exs [hunk 2]
```diff
@@ -1,9 +1,9 @@
-defmodule AdventOfCode2019.MixProject do
+defmodule AdventOfCode.MixProject do
   use Mix.Project
 
   def project do
     [
-      app: :advent_of_code_2019,
+      app: :advent_of_code,
       version: "0.1.0",
       elixir: "~> 1.7",
```
Comment (mhanberg) [inline mix.exs:8]:
```suggestion
      elixir: "~> 1.9",
```

**Comment by lytedev** [comment]
Changes made!

**Comment by mhanberg** [comment]
Thanks!

File: lib/advent_of_code/input.ex [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+  import Logger
+
+  @doc """
+  Retrieves the specified input for your account. If the input is not in your
+  cache, it will be retrieved from the server if `allow_network?: true` is
+  configured and your cookie is setup.
+  """
+  def get!(day, year \\ nil)
+  def get!(day, nil), do: get_input!(day, default_year())
```
Comment (mhanberg) [inline lib/advent_of_code/input.ex:18]: this this be just `get!(day, default_year())`? @lytedev

Comment (lytedev) [reply lib/advent_of_code/input.ex:18]: Oops yes.

Comment (mhanberg) [reply lib/advent_of_code/input.ex:18]: No worries, I've got it fixed locally 👍

**Pull request merged**